provider/github/client.go

```diff
@@ -443,3 +453,47 @@ func handleAPIError(resp *github.Response, err error, msg string) error {
 		msg,
 	)
 }
+
+// ValidateTokenPermissions checks that client has necessary permissions required by lookout
+// returns error if any is missed
+func ValidateTokenPermissions(client *Client) error {
+	// actually we don't need ALL repo scope but we ask for it in documentation
```

Comment:
If we know we don't it would also make sense to update the docs.

Reply:
I'm not sure why we say we require "repo" access.

Looks like it was introduced here https://github.com/src-d/lookout/pull/398
@dpordomingo do you remember why we require `repo` instead of `repo:status`?

Reply:
figured. we need it for private repos. I'll remove this comment from the code.

Reply:
ups, sorry, but blame plane landed in @carlosms :trollface: 
https://github.com/src-d/lookout/pull/294/commits/62c64c107e4854b7a7dfd4f990eaa726a8af2b5b#diff-76e731333fb756df3bff5ddb3b731c46R38